lang/cc65/Portfile

```
@@ -12,43 +14,40 @@ long_description \
                 cc65 is a complete cross development package for 65(C)02 \
                 systems, including a powerful macro assembler, a C compiler, \
                 linker, librarian and several other tools.
+license         CC-BY-SA 
 
-# Per http://www.cc65.org/ discontinued support in 2013, moved to github
-# No version tags on github yet
 homepage        http://cc65.github.io/cc65
-distfiles       cc65-sources-${version}.tar.bz2
-use_bzip2       yes
+distfiles       V${version}.tar.gz
 
-master_sites    http://cc65.oldos.net/ \
-                http://bj.spline.de/cc65/ \
-                ftp://ftp.musoftware.de/pub/uz/cc65/ \
-                ftp://ftp.musoftware.de/pub/uz/cc65/old/
+master_sites    git:https://github.com/cc65/cc65/archive
```

No need for separate master_sites when using github.setup but do use github.tarball_from:

```suggestion
github.tarball_from archive
```

---

done!